templates/components/itilobject/timeline/timeline_item_header_badges.html.twig

```diff
@@ -31,14 +31,23 @@
  # ---------------------------------------------------------------------
  #}
 
-<span class="badge text-wrap d-none d-md-block">
+{% set timeline_display_date = session('glpitimeline_date_format') %}
+<span class="badge user-select-auto text-wrap d-none d-md-block">
    {% set date_span %}
-      <span title="{{ date_creation|formatted_datetime }}"
-            data-bs-toggle="tooltip" data-bs-placement="bottom">
+      <span
+         {% if timeline_display_date == constant('Config::TIMELINE_RELATIVE_DATE') %}
+            title="{{ date_creation|formatted_datetime }}"
+            data-bs-toggle="tooltip" data-bs-placement="bottom"
+         {% endif %}
+      >
          <i class="far fa-clock me-1"></i>
-         <a href="#{{ anchor }}">
-            {{ date_creation|relative_datetime }}
-         </a>
+         {% if timeline_display_date == constant('Config::TIMELINE_RELATIVE_DATE') %}
+            <a href="#{{ anchor }}">
```

Comment:
`<a href="#{{ anchor }}">` should probably be places outside if/else block to apply on both date formats

Reply:
done